### PR TITLE
Feature/tr 4778/extend ags send error log message payload with delivery execution

### PR DESCRIPTION
**Pull request by pnal**
Extending critical log entry payload in order to be able to find related execution.

### CODE REVIEW

File: model/events/LtiAgsListener.php [hunk 1]
```diff
@@ -59,6 +60,7 @@ public function onDeliveryExecutionStart(DeliveryExecutionCreated $event): void
             $taskQueue = $this->getServiceLocator()->get(QueueDispatcherInterface::SERVICE_ID);
             $taskQueue->createTask(new SendAgsScoreTask(), [
                 'registrationId' => $user->getRegistrationId(),
+                'deliveryExecutionId' => $deliveryExcecution->getIdentifier(),
```
Comment (Karol-Stelmaczonek) [inline model/events/LtiAgsListener.php:63]: maybe check if $deliveryExcecution is an object (using instanceof) before using mehtod. Better safe than sorry.

Comment (Karol-Stelmaczonek) [reply model/events/LtiAgsListener.php:63]: or if you are 100% sure then dont :D

**Review by Karol-Stelmaczonek — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by andreluizmachado — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**